server/server/src/billing/stripe_client.rs

```diff
@@ -0,0 +1,273 @@
+use crate::billing::stripe_client::SimplifiedStripeError::{
+    CardDeclined, InvalidCreditCard, Other,
+};
+
+use crate::{ServerState, StripeDeclineCodeCatcher, StripeKnownDeclineCode};
+use lockbook_models::api::{CardDeclineReason, CardRejectReason};
+use log::{error, info};
+use std::fmt::Debug;
+use stripe::{
```

Comment:
Imo, same thing goes for here, even though we are in the `stripe_client` file. Drop the stripe imports and use the mod path.